mscore/editelement.cpp

```diff
@@ -94,6 +94,9 @@ void ScoreView::startEditMode(Element* e)
             }
       if (score()->undoStack()->active())
             score()->endCmd();
+      // Restart edit mode to reinit edit values
+      if (editMode())
+            changeState(ViewState::NORMAL);
```

Comment:
Interestingly, I have actually done a similar thing for text editing in #5570 (see [this diff](https://github.com/musescore/MuseScore/pull/5570/files#diff-a7fddef1509bdc1ec91a110d8e6f5136R95-R98), at the bottom of the page). With that PR having been merged, could we perhaps unite these two changes? Probably replacing `textEditMode()` check by just `editMode()` should be enough. I am not sure though whether this should happen before or after these `endCmd()` lines (which shouldn't be normally executed anyway?).